src/components/header/Header.css

```diff
@@ -0,0 +1,47 @@
+
+.header_container {
+    background-color: #0c6aa7;
+    width: 100%;
+    height: 60px;
+    position: fixed;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 10px;
+    left: 0;
+}
+
+::placeholder {
```

Comment:
Target placeholder more specifically. This will modify all placeholders on page. If you still want to modify all of them, assign them same class and then target it with that class.